worklets/Overview.bs

```diff
@@ -262,49 +265,83 @@ is empty; it is populated when the user agent chooses to create its {{WorkletGlo
 A {{Worklet}} has a list of the <dfn>worklet's resolved module URLs</dfn>. Initially this list is
 empty; it is populated when module scripts resolved.
 
+A {{Worklet}} has a <dfn>fetched modules map</dfn>. This is a map of <a>fetch</a> requests to values
+that are a <a>fetch</a> responses. Access to this map should be thread-safe.
+
+The <a>fetched modules map</a> exists to ensure that {{WorkletGlobalScope}}s created at different
+times contain the same set of scripts and have the same behaviour. The creation of additional
+{{WorkletGlobalScope}}s should be transparent to the author.
+
 When the <dfn method for=Worklet>import(|moduleURL|)</dfn> method is called on a {{Worklet}} object,
 the user agent <em>must</em> run the following steps:
     1. Let |promise| be <a>a new promise</a>.
 
-    2. Run the following steps <a>in parallel</a>:
-        1. Let |resolvedModuleURL| be the result of <a>resolving</a> the |moduleURL| relative to the
-            <a>API base URL</a> specified by the <a>entry settings object</a> when the method was
-            invoked.
+    2. Let |resolvedModuleURL| be the result of <a>resolving</a> the |moduleURL| relative to the
+        <a>API base URL</a> specified by the <a>entry settings object</a> when the method was
```

Comment:
[Don't use entry settings object.](https://readable-email.org/list/public-script-coord/topic/multiple-globals-and-you) Use current settings object. Probably the same, in a worklet...